Sources/SwiftSyntaxMacros/MacroExpansionDiagnosticMessages.swift

```diff
@@ -68,3 +68,20 @@ public struct MacroExpansionFixItMessage: FixItMessage {
     self.message = message
   }
 }
+
+/// The message of a note that is specified by a string literal
+///
+/// This type allows macro authors to quickly generate note messages based on
+/// a string. For any non-trivial note messages, it is encouraged to define a
+/// custom type that conforms to `NoteMessage`.
+public struct MacroExpansionNoteMessage: NoteMessage {
+  public var message: String
+
+  public var fixItID: SwiftDiagnostics.MessageID {
```

Comment:
The name of this property has been updated in the [Rename NoteMessage.fixItID to noteID](https://github.com/apple/swift-syntax/pull/2264) pull request. I believe that due to this change, the CI is failing. 

```suggestion
  public var noteID: SwiftDiagnostics.MessageID {
```